src/app/frontend/logs/component.js

```diff
@@ -92,8 +100,16 @@ export class LogsController {
     /** @private {!ui.router.$stateParams} */
     this.stateParams_;
 
-    /** @export {!kdUiRouter.$transition$} - initialized from resolve */
+    /** @export {!kdUiRouter.$transition$} */
     this.$transition$;
+
+    /** @export {boolean} */
+    this.isFollowing = false;
```

Comment:
All other buttons store the state in the logsService. Not sure if this service is very beneficial, but until we refactor I would suggest to keep it consistent